### PR TITLE
PR: Last Stretch for RC2

### PR DESCRIPTION
**Pull request by KelSolaar**

### CODE REVIEW

File: opencolorio_config_aces/config/studio/generate/resources/OpenColorIO-Config-ACES CG and Studio Transforms - v1.0.0-rc2 - Studio Config - Mapping.csv [hunk 1]
```diff
@@ -10,7 +10,9 @@ Ordering,Colorspace,Legacy,ACEStransformID,CLFtransformID,Interface,BuiltinTrans
...
 204,Input - ARRI - Linear - ARRI Wide Gamut 4,FALSE,,urn:aswf:ocio:transformId:1.0:ARRI:Input:Linear_ARRI_Wide_Gamut_4_to_ACES2065-1:1.0,ColorSpace,,lin_awg4,scene-linear,
 204,Input - ARRI LogC4,FALSE,,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC4_to_ACES2065-1:1.0,ColorSpace,,,log,
+210,Input - Blackmagic - Blackmagic Film Wide Gamut (Gen 5),FALSE,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:BMDFilm_WideGamut_Gen5_to_ACES2065-1:1.0,ColorSpace,,,log,Blackmagic
 210,Input - Blackmagic - Curve - DaVinci Intermediate,FALSE,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:DaVinci_Intermediate_Log-Curve_to_Linear:1.0,NamedTransform,,,log,Blackmagic
+210,Input - Blackmagic - Curve - DaVinci Intermediate Log,FALSE,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:BMDFilm_Gen5_Log-Curve_to_Linear:1.0,NamedTransform,,,log,Blackmagic
 210,Input - Blackmagic - DaVinci Intermediate - DaVinci Wide Gamut,FALSE,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:DaVinci_Intermediate_WideGamut_to_ACES2065-1:1.0,ColorSpace,,,log,Blackmagic
```
Comment (doug-walker) [inline opencolorio_config_aces/config/studio/generate/resources/OpenColorIO-Config-ACES CG and Studio Transforms - v1.0.0-rc2 - Studio Config - Mapping.csv:16]: This file is wrong, there were some errors in the spreadsheet.  I just made the following corrections to the spreadsheet:

Row 193, Column A, change 
"Input - Blackmagic - Curve - DaVinci Intermediate Log"  to 
"Input - Blackmagic - Curve - Blackmagic Film (Gen 5)"

Row 194, Column A, change 
"Input - Blackmagic - Linear - DaVinci Wide Gamut" to 
"Input - Blackmagic - Linear - Blackmagic Wide Gamut (Gen 5)"

Row 194, Column G, change "urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:Linear_DaVinci_WideGamut_to_ACES2065-1:1.0" to 
"urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:Linear_BMD_WideGamut_Gen5_to_ACES2065-1:1.0"

If you pull down the latest csv, it should fix it.

**Review by doug-walker — CHANGES_REQUESTED** [review]

**Comment by doug-walker** [comment]
I pulled down the branch, ran "invoke update-mapping-file-studio" to update the csv file and generated a studio config.  This fixed the BMD issue, and updated the categories (using 'file-io' rather than vendor labels).  I tested this in Maya 2023 update 2 and it worked well.  

A few remaining issues I found:
1)  ACES2065-1 still does not have a category in the config, even though it is listed properly in the Studio spreadsheet.
2)  Minor, but I'm thinking the family for "Rec.709 - Curve" should be changed from "Utility/OCIO" to "Utility/ITU" so that it sits alongside the "Camera Rec.709" color space.  And perhaps "ST-2084 - Curve" should do likewise?
3)  The default role should either be removed or changed to "ACES2065-1".
4)  The description in the 2.1 config is the same as the 2.0 config (should use the "2.1" numbering).

For the CG config, here are the items I found:
5)  The 2.1 version is missing the ACES 1.3 Gamut Compression.
6)  I'm wondering if we should drop the OCIO family hierarchy for the CG config.  It makes reaching the most common spaces a bit more difficult in the hierarchical menus, and the extra organization/filtering is not needed here.
7)  I'm wondering if we should hide the Display color spaces for the CG config (make them inactive).  They are not essential and may add confusion.

**Comment by KelSolaar** [comment]
I should have addressed all the notes but 2, 6 and 7.

**Comment by doug-walker** [comment]
Thank you Thomas!  I pulled down the latest commits and built the updated configs.  All my comments above have now been addressed!

The last problem I've found is with the Named Transforms.  These should be converting from the non-linear space to the linear space, but they are doing the opposite in the current configs.  The camera log curve CLFs have the transform in the correct direction, and the description in the config is in the correct direction, but the actual transform is opposite.  It should use "transform" rather than "inverse_transform".  

For the monitor transfer function curves, the CLFs are actually backwards.  The easiest fix for those would be to change "transform" to "inverse_transform" in the config.  The description will be backwards, but that would be ok for RC2.  I could do a PR to fix those CLFs later.

**Comment by doug-walker** [comment]
And it looks like we need to swap the matte_paint and texture_paint roles to align with Nuke's input file mapping.

**Comment by KelSolaar** [comment]
I swapped the direction of the named transform in the code, and swapped the roles.

**Comment by doug-walker** [comment]
I pulled the commits and rebuilt the Studio config.  I confirm that the roles are swapped, as discussed.  And the Named Transforms are all fixed except the last one "ST-2084 - Curve".  That one has a slightly different syntax (does not use a GroupTransform in the Yaml) and was not adjusted for some reason.

**Comment by KelSolaar** [comment]
Good catch @doug-walker, it is because it comes as a CLF but because it cannot be serialised in the config, we use the builtin instead, I needed to update that code path too. Should be done now.

**Comment by doug-walker** [comment]
OK, your latest push seems to have added an inverse_transform but didn't remove the original transform:
```
  - !<NamedTransform>
    name: ST-2084 - Curve
    aliases: [st_2084_crv, crv_dolbypq_10000]
    description: Convert linear nits/100 to SMPTE ST-2084 (PQ) full-range
    family: Utility
    categories: [file-io]
    encoding: hdr-video
    transform: !<BuiltinTransform> {style: CURVE - LINEAR_to_ST-2084}
    inverse_transform: !<BuiltinTransform> {style: CURVE - LINEAR_to_ST-2084}
```

**Comment by KelSolaar** [comment]
`/sad_panda_face`

**Comment by KelSolaar** [comment]
Should be good for real, I checked! :)

**Review by doug-walker — APPROVED** [review]
Victory!  Everything looks good.  Let's do RC2!!

**Comment by KelSolaar** [comment]
💥 🍾

**Review by doug-walker — APPROVED** [review]

**Pull request merged**